software/util/reorg.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+"""A Tool to help maintain the set of ttl files in schema.org.
+
+This helps reformatting, merging and splitting ttl files along
+semantic axis.
+"""
+
+from collections.abc import Sequence
+from rdflib import Graph, Namespace
+import logging
+import sys
+import argparse
+
+def SchemaOrgGraph():
+  """Creates a graph with usual namespaces set as we want in schemaorg."""
+  g = Graph()
+  g.bind("dc", Namespace("http://purl.org/dc/terms/"), replace=True)
+  return g
+
+def GraphsAreIdentical(g1, g2):
+  only_in_g1 = g1 - g2
+  only_in_g2 = g2 - g1
+  # TODO: print sampled diffs if any.
+  return len(only_in_g1) == 0 and len(only_in_g2) == 0
+
+def GraphIsIncludedIn(g_subset, g_universe):
+  only_in_subset = g_subset - g_universe
+  # TODO: print sampled diffs if any.
+  return len(only_in_subset) == 0
+
+def Lint(args):
```

Comment:
I would pass the arguments explicitly, i.e. output_filename and files so the function can be easily called from Python code/unit-tests.